docs/v2.3/releaseNotes.yml

```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
+          It now gives the correct error message, instructing the user on how to specify which
+          service the intercept should use.
+      - type: bugfix
+        title: Traffic-manger determine's service subnet in own namespace
```

Comment:
This is a little hard to parse, maybe `Traffic Manager creates service in its own namespace to determine service subnet`